Modules/timemodule.c

```diff
@@ -2051,34 +2051,51 @@ PyInit_time(void)
 static int
 pysleep(_PyTime_t secs)
 {
-    _PyTime_t deadline, monotonic;
```

Comment:
Interlaced ifdef/else made sense when code was in common between Unix and Windows. With your change, it seems like the Windows code is completely different and so a single #ifdef/else for the whole function should be enough. Put the Windows code at the end ;-)

Reply:
After my commit, yes i realized that Windows part is totaly separated, so i can merge the full code in a single `#ifndef MS_WINDOWS ... #else ... #endif` block. I will improve it.